test/e2e/e2e.bats

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bats
+
+
+# Set up test environment
+
+source ./test/helper/helper.sh
+
+
+# Deploy task to test environment
+kubectl apply -f templates/task-sc.yaml
+
+
+# Define test case
+read -p "Enter source registry URL: " SOURCE_REGISTRY
+read -p "Enter destination registry URL: " DESTINATION_REGISTRY
+read -p "Enter source registry username: " SOURCE_AUTH_USERNAME
+read -p "Enter source registry password: " -s SOURCE_AUTH_PASSWORD
+echo  "Source registry Credentials are taken as input"
+
+read -p "Enter destination registry username: " DESTINATION_AUTH_USERNAME
+read -p "Enter destination registry password: " -s DESTINATION_AUTH_PASSWORD
+echo  "Destination registry Credentials are taken as input"
```

Comment:
The configuration needed for testing should come from the environment variables, for instance:
- https://github.com/otaviof/tkn-buildpacks-task/blob/02293f1720a8f59a3051dad98e2e8b99d27fe889/Makefile#L10-L14
- https://github.com/otaviof/tkn-buildpacks-task/blob/02293f1720a8f59a3051dad98e2e8b99d27fe889/test/e2e/e2e.bats#L40-L42

Then running the tests with alternative settings becomes:
```bash
$ make test-e2e VARIABLE="new-value"
```

Where `${VARIABLE}` will end up on the scripts we need.